blueprint/api.apib

```diff
@@ -1237,6 +1237,7 @@ This resource identifies a relationship between a User and a Skill. For example,
         + data
             + Include (User Resource Identifier)
             + attributes (object)
+                + name: `Test` (string, optional)
```

Comment:
I would also specify that it's read-only in the documentation and that it's simply a combination of `first_name` and `last_name`.